modules/spdlog/1.12.0/patches/add_build_file.patch

```diff
@@ -0,0 +1,21 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,18 @@
++load("@rules_cc//cc:defs.bzl", "cc_library")
++
++licenses(["notice"])  # Apache 2
++
++package(
++    default_visibility = ["//visibility:public"],
++)
++
++cc_library(
++    name = "spdlog",
++    hdrs = glob([
++        "include/**/*.cc",
```

Comment:
There are no `.cc` files in the `include` directory of spdlog, could we remove this line? Otherwise an error occurs when the `--incompatible_disallow_empty_glob` flag is set

Reply:
I just submitted #1218 removing it from the previous version